.github/workflows/build.yml

```diff
@@ -156,7 +156,7 @@ jobs:
         env:
           PACK_BUILD: ${{ github.run_number }}
         shell: powershell
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
         if: matrix.config != 'windows-lcow'
         with:
           name: pack-${{ matrix.os }}
```

Comment:
Checking the [migration](https://github.com/actions/upload-artifact/blob/main/docs/MIGRATION.md) guide to v4, I think it is safe to do the update, we are already generating a different name for the binary according to the platform